pkg/generator/common/types.go

```diff
@@ -0,0 +1,12 @@
+package common
```

Comment:
May be keep all the common types to generator package instead of a common package? And move all to separate packages inside.

Reply:
moved